bluesky/time.go

```diff
@@ -1,17 +1,36 @@
 package bluesky
 
-import "time"
+import (
+	"strings"
+	"time"
+)
 
 var timeFormat = "2006-01-02T15:04:05.999999999Z"
 
 func ParseTime(str string) (time.Time, error) {
+	if strings.HasSuffix(str, "Z") {
+		return parseZulu(str)
+	} else {
+		return parseWithOffset(str)
```

Comment:
Weirdly - I've not received any events over the firehose that use an offsetted time. Did this come up in testing with the Indigo implementation ?

Reply:
Yeah, that happened to me with the indigo server. I’m in UTC+2 right now, so at least some of the timestamps ended with `+02:00`.